core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -140,25 +150,28 @@ object ReplicaVerificationTool extends Logging {
     val maxWaitMs = options.valueOf(maxWaitMsOpt).intValue
     val initialOffsetTime = options.valueOf(initialOffsetTimeOpt).longValue
     val reportInterval = options.valueOf(reportIntervalOpt).longValue
+  }
+
+  def main(args: Array[String]): Unit = {
+    val opts = new ReplicaVerificationToolOptions(args)
+
     // getting topic metadata
     info("Getting topic metadata...")
-    val brokerList = options.valueOf(brokerListOpt)
-    ToolsUtils.validatePortOrDie(parser, brokerList)
 
     val (topicsMetadata, brokerInfo) = {
-      val adminClient = createAdminClient(brokerList)
-      try ((listTopicsMetadata(adminClient), brokerDetails(adminClient)))
+      val adminClient = createAdminClient(opts.bootstrapServer)
+      try (listTopicsMetadata(adminClient), brokerDetails(adminClient))
       finally CoreUtils.swallow(adminClient.close(), this)
     }
 
-    val topicIds = topicsMetadata.map( metadata => metadata.name() -> metadata.topicId()).toMap
+    val topicIds = topicsMetadata.map(metadata => metadata.name() -> metadata.topicId()).toMap
 
     val filteredTopicMetadata = topicsMetadata.filter { topicMetaData =>
-      topicsIncludeFilter.isTopicAllowed(topicMetaData.name, excludeInternalTopics = false)
+      opts.topicsIncludeFilter.isTopicAllowed(topicMetaData.name, excludeInternalTopics = false)
     }
 
     if (filteredTopicMetadata.isEmpty) {
-      error(s"No topics found. $topicsIncludeOpt if specified, is either filtering out all topics or there is no topic.")
+      error(s"No topics found. ${opts.topicWhiteListOpt} if specified, is either filtering out all topics or there is no topic.")
```

Comment:
Is it intended to change from `topicsIncludeOpt` to `topicWhiteListOpt` ?

Reply:
Oh no, this modification is accidentally introduced by merging; Fixed.